include/nccl-headers/nvidia/net.h

```diff
@@ -24,11 +24,14 @@ typedef enum {NCCL_INIT=1, NCCL_COLL=2, NCCL_P2P=4, NCCL_SHM=8, NCCL_NET=16, NCC
 
 typedef void (*ncclDebugLogger_t)(ncclDebugLogLevel level, unsigned long flags, const char *file, int line, const char *fmt, ...);
 
+/* disable include sorting */
```

Comment:
The files in include/nccl-headers/nvidia and include/nccl-headers/neuron must not be touched.  They are synced from their upstreams directly and I don't want to add any overheads.  We need to find another way to do this.

Reply:
Current revision takes care to exclude those files from anything.